### ISSUE TITLE
Text annotation with gplotly()

### ISSUE THREAD

**Issue by sreejithmunthikodu** [UBC-MDS/DSCI_532_group_205_movies_R#6]
I am able to do text annotation in the lower plot with respective movie names. I used library ggrepel. It worked great as a static plot. However, while wrapping over ggplotly(), it doesn't work. It seems ggrepel is not yet implemented in ggplotly(). I will keep exploring other options.

**Comment by techrah** [comment]
Thanks for looking into this @sreejithmunthikodu. Please check out the solution suggested by Firas and see how far you can get with this.

https://plot.ly/r/text-and-annotations/

**Comment by sreejithmunthikodu** [comment]
I tried it. But the link above is for plotly objects. We need to add text on top of a hex-plot. Issue is hex-plot is only in ggplot(). Now we need to combine two different objects into a single plot. plotly and ggplotly. So far I am unsuccessful in merging them. The usual ggplot p + new plot syntax isn't working. I'm still on it.

**Comment by sreejithmunthikodu** [comment]
I was able to combine the plotly and ggplotly objects.
However, now that everything is working, there is the problem of overlapping texts. I tried different text angles. But whatever angle I set, there are some selection combinations that results in overlapping of some of the text labels in the scatterplot.
I have gone through the different attributes of adding annotations (https://github.com/plotly/plotly.js/blob/master/src/components/annotations/attributes.js). But I dont think there is a way to automatically prevent overlapping.

If this doesn't resolve, we may have to go back and use different colors for different movies.
